Jenkinsfile

```diff
@@ -1,8 +1,6 @@
-@Library('pipeline-library@pull/784/head') _
-
 pipeline {
     agent {
-        label 'jdk17'
+        label 'jdk17 || linux-arm64'
```

Comment:
Build fails on infra.ci.jenkins.io with the following error:

```
08:26:45  * What went wrong:

08:26:45  Could not determine the dependencies of task ':javadoc'.

08:26:45  > Failed to calculate the value of task ':compileJava' property 'javaCompiler'.

08:26:45     > Cannot find a Java installation on your machine matching this tasks requirements: {languageVersion=17, vendor=any, implementation=vendor-specific} for LINUX on aarch64.

08:26:45        > No locally installed toolchains match and toolchain download repositories have not been configured.
```

=> looks like we need to amend this pipeline with a `tool` directive

Reply:
Note: add a comment about arm, already done in the Dockerfile via platforms

Reply:
And an issue to build with arm on ci.jenkins.io too like the platform used in prod.